pkg/machines/components/notification/inlineNotification.jsx

```diff
@@ -76,21 +76,17 @@ InlineNotification.propTypes = {
 
 export const Alert = ({ onDismiss, text, textId, detail }) => {
```

Comment:
Maybe it would instead make sense to rename this as `WarningAlert`, to avoid confusing this with PF-react's class? Also, the [documentation](https://rawgit.com/patternfly/patternfly-react/gh-pages/patternfly-3/index.html?selectedKind=patternfly-react%2FWidgets%2FAlert&selectedStory=Alert%20types&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) shows an `Alert` without a nested `InlineNotification` element -- is that still actually necessary? If not, it may make sense to just drop this class (as then the only thing it really does is to set `type=warning`), and just use PF-React's `Alert` as -is?

Reply:
Our InlineNotification builds the Alert body, which is not plain string.
It has some message string, then a href link tagged as `show more` which when clicked will show the notification details.
So yes, the InlineNotification class is necessary. The docs example has notification body a plain string, that's why they use PF Alert without extra things.

Reply:
I 'll repost to fix the naming.

Reply:
Makes sense, thanks!  So perhaps `NotificationAlert`? (<bikeshedding>)